debug/src/debug.js

```diff
@@ -58,6 +60,8 @@ export function initDebug() {
 			Expected a valid HTML node as a second argument to render.
 			Received ${parentNode} instead: render(<${vnode.type.name || vnode.type} />, ${parentNode});
 		`);
+
+		if (oldRoot) oldRoot(vnode, parentNode);
```

Comment:
Ohh this is a good one! Never thought about options page in case of render errors before 👍💯

Reply:
😁